### PR TITLE
add section for intro to image analysis

### PR DESCRIPTION
**Pull request by alessandrofelder**
## Description

**What is this PR**
Adds slides for image analysis and napari intro sections.

### CODE REVIEW

File: slides/intro-napari.qmd [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+```{.bash}
+napari
+```
+
+![](img/blank-napari-ui.png){alt="A screenshot of the default Napari user 
+interface"}
+
+## Opening images
+
+`File > Open Files(s)`, then navigate to `calcium imaging folder` and open `translation1_00001_ce.tif`
```
Comment (adamltyson) [inline slides/intro-napari.qmd:12]: Is this going to be on the shared drive?

File: slides/what-is-an-image.qmd [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+## Image data type
+
+:::: {.columns}
+
+::: {.column width=45%}
+The other key feature of an image array is its 'data type' - this controls which 
+values can be stored inside of it.
+``` {.python}
+
```
Comment (adamltyson) [inline slides/what-is-an-image.qmd:110]:
```suggestion
```

File: slides/processing-images.qmd [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+print(pixels_in_each_region)
+```
+![](img/regionprops.png){alt="A list of pixel counts for each region of the example image"}
+
+## Key points
+
+* Segmentation can be broadly split into ‘semantic segmentation’ and ‘instance segmentation’. Semantic segmentation assigns pixels to specific classes (like neuron or background), while instance segmentation assigns pixels to individual ‘instances’ of a class (like individual neurons).
+* Segmentations are represented in the computer in the same way as standard images. The difference is that pixel values represent classes or instances, rather than light intensity.
+* Napari uses Labels layers for segmentations.
+* Segmentation is helpful for analysis.
```
Comment (adamltyson) [inline slides/processing-images.qmd:53]: This is a bit wordy for slides?

**Review by adamltyson — APPROVED** [review]
very nice, thanks!

File: slides/intro-napari.qmd (hunk 1 above)
Comment (alessandrofelder) [reply slides/intro-napari.qmd:12]: yep

File: slides/processing-images.qmd (hunk 3 above)
Comment (alessandrofelder) [reply slides/processing-images.qmd:53]: I'll shorten.
